packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -255,6 +286,20 @@ export class SnapInterfaceController extends BaseController<
     });
   }
 
+  /**
+   * Update the type of content in an interface.
+   *
+   * @param id - The interface id.
+   * @param contentType - The type of content.
+   */
+  updateInterfaceContentType(id: string, contentType: ContentType) {
```

Comment:
Wondering if this is overkill for now. Since we strictly need this for static notifications, this adds a lot of complexity that isn't currently needed. WDYT?

Reply:
It's required to reliably mark an interface's content type. At various points in the system we accept interface id's instead of the JSX content itself. This is meant to address those situations. I didn't want to expose the concept of `contentType` to the end user and thus chose not to try and force it in the `createInterface` method.

Reply:
Agreed that we shouldn't allow the user to modify this. My comment was more regarding that we only need this information for notifications right now, so we don't need to solve it for all other types of interfaces yet.

Reply:
Ohh, that's valid. Hmm I'll make some changes.